src/main/java/gbot/Ui.java

```diff
@@ -0,0 +1,44 @@
+package gbot;
+
+/**
+ * The user interface of the chatbot.
+ *
+ * @author Gallen Ong
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________";
+
+    /**
+     * Prints a message in between two lines.
+     *
+     * @param message The message to be printed.
+     */
+    public static void print(String message) {
+        System.out.println(LINE);
+        System.out.println(message);
+        System.out.println(LINE);
+    }
+
+    /**
+     * Prints the initial greeting to user.
+     */
+    public void ask() {
```

Comment:
You can consider renaming the name of this method to be more descriptive. "ask" might not be the most accurate description of what this method is doing